examples/table-queries-select-all/table_queries_select_all.bal

```diff
@@ -0,0 +1,44 @@
+import ballerina/io;
+
+// This `record` type represents a person.
+type Person record {
+    int id;
+    int age = -1;
+    float salary;
+    string name;
+    boolean married;
+};
+
+// This is the main function.
+public function main() {
+    string queryStmt = "";
+
+    // These are a few sample values that represent different `Person` records.
+    Person p1 = { id: 1, age: 25, salary: 1000.50, name: "jane", married: true };
+    Person p2 = { id: 2, age: 26, salary: 1050.50, name: "kane", married: false };
+    Person p3 = { id: 3, age: 27, salary: 1200.50, name: "jack", married: true };
+    Person p4 = { id: 4, age: 28, salary: 1100.50, name: "alex", married: false };
```

Comment:
Incorrect formatting. We shouldn't have space after `{` and before `{`. Check all the examples